src/mappings/tokenLockWallet.ts

```diff
@@ -55,3 +57,19 @@ export function handleRevokeTokenDestinations(
   tokenLockWallet.tokenDestinationsApproved = false;
   tokenLockWallet.save();
 }
+
+export function handleTokenDestinationsApproved(
```

Comment:
Well yes, I think we could by forcing the type. The main difference would be that the typed event wouldn't be the exact same received by the handler, and thus reusing the handler wouldn't work (we'd still need to have them as separate handlers, otherwise they won't be properly recognized by graph-node, but we could make it so that we force cast the type so we can simply call the other handler, or make a generic function that only receives basic inputs to generate the changes, but to be honest, that would probably be more lines of code than it's worth it due to the small footprint of the handlers 😅 )